images/Homepage_files/homepage.css

```diff
@@ -0,0 +1,394 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@400;700&family=Outfit:wght@100;400&display=swap');
+
+* {
```

Comment:
move common styles into a different folder such as 'main.css', do not repeat yourself